server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/components/_collapse.scss

```diff
@@ -0,0 +1,86 @@
+/*!
```

Comment:
Please do not add a bang `!` it gets copied into the generated output.

Reply:
Fixed, thanks

Reply:
I should add that this is the default behavior with intellij (and/or the copyright plugin we use), and I could not find a toggle to turn it off.

If there's a lint we can add to catch this, perhaps we should consider adding it.